yarn-project/end-to-end/src/e2e_cross_chain_messaging/l2_to_l1.test.ts

```diff
@@ -140,7 +140,7 @@ describe('e2e_cross_chain_messaging l2_to_l1', () => {
       await aztecNodeAdmin.setConfig({ minTxsPerBlock: 2 });
 
       const [noMessageReceipt, messageReceipt] = await Promise.all([
-        contract.methods.set_constant(Fr.random()).send().wait(),
+        contract.methods.emit_nullifier(Fr.random()).send().wait(),
```

Comment:
Did this change such that I can nuke the `set_constant` function from the `Test` contract. Here the `set_constant` was used only as an arbitrary function that does not emit l2 to l1 message.